alfalfa_worker/jobs/step_run_base.py

```diff
@@ -186,6 +207,39 @@ def stop(self) -> None:
         super().stop()
         self.set_run_status(RunStatus.STOPPING)
 
+        # Clear current values from the database when the simulation is no longer running
+        if not self.skip_stop_db_writes:
```

Comment:
here in order to enable certain unit tests that don't need the db. not the prettiest :(